### ISSUE TITLE
Only voice when playing mp4 video in chrome

### ISSUE THREAD

**Issue by zzbslayer** [videojs/video.js#5307]
## Description
Similar to this issue [issue#1590](https://github.com/videojs/video.js/issues/1590)
Is there any other way to solve this problem **except encoding the video for web** mention in the above issue?
There's only voice when playing mp4 video in chrome. The audio goes smoothly but the screen is black. But It runs very well in safari.

## Sources
[jsbin](https://jsbin.com/punojecunu/edit?html,output)

## Steps to reproduce
Just click the jsbin link.
In chrome, the video is played only with audio.
In safari, it works perfectly.

### videojs-contrib-hls version
- "videojs-contrib-hls": "^5.14.1"

### videojs version
- "video.js": "^7.0.5"

### Browsers
Chrome 67.0.3396.99

### Platforms
macOS High Sierra 10.13.1

**Comment by ldayananda** [comment]
I just tried with Chrome Version 67.0.3396.99 with MacOS Sierra 10.12.6 and was able to see both audio and video. Also tested on a couple of High Sierra machines and saw audio and video. Are you able to test on another machine to confirm that browser-platform combination is the issue?

**Comment by zzbslayer** [comment]
It's so strange. My friends couldn't play the video either yesterday.
But now there's both audio and video, too. 
I guess there might be some problems about the nginx server. 
Really thanks. Maybe this issue can be closed.

**Comment by victoryboss** [comment]
There are something wrong with your mp4 file . 
Please fomat the mp4 file using H264.

**Comment by davide24** [comment]
I have the same problem.. I did a small video with my mobile phone and under Chrome i cannot see the video but hear the audio but it works under Safari.. (Ubuntu -> Chrome)

**Comment by gkatsev** [comment]
It's possible your video was created using HEVC which Safari can play but Chrome can't.

**Comment by bertrandg** [comment]
hi @zzbslayer @davide24 @gkatsev 

I've seen that many browsers can't play mp4 videos using hevc/h265 codec:
https://caniuse.com/hevc

But do you know if there's a native way to detect if video is not played correctly (audio only) on frontend side to alert end users?

**Comment by gkatsev** [comment]
I've generally seen playback fail completely if the video codec isn't supported, in which case you'd get an error event. I'm not sure if there's an easy way to know whether it's playing audio-only even if it shouldn't be.

**Comment by mister-ben** [comment]
Check `player.videoHeight() > 0` perhaps

**Comment by bertrandg** [comment]
Yeah nice hack idea, I'm gonna try it soon and let you know there.
Thanks

**Comment by bertrandg** [comment]
Just tried your solution, works like a charm! 🥇

```
const video = document.querySelector('video');

video.addEventListener('loadedmetadata', (event) => {
  if(event.target.videoHeight === 0) {
    console.log('cant play video..');
  }
});
```
